### PR TITLE
add EMAIL_URL & ACCOUNT_MANAGEMENT_VISIBLE environment variables

### PR DESCRIPTION
**Pull request by ubergesundheit**
This pull request adds:

Handling of `EMAIL_URL` environment variable for SMTP configuration. I couldn't test this as I have some problems with gandi being blocked on scaleway but wanted to get your attention ahead of time. I'll ping you here once I've made sure this works.

`ACCOUNT_MANAGEMENT_VISIBLE` variable for enabling/disabling links to features requiring a valid email set up.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
I successfully sent mails using this configuration! Works for me ;)

File: meine_stadt_transparent/settings/__init__.py [hunk 1]
```diff
@@ -53,6 +53,9 @@
         "MAILJET_SECRET_KEY": env.str("MAILJET_SECRET_KEY"),
     }
     EMAIL_BACKEND = "anymail.backends.mailjet.EmailBackend"
+else:
+    EMAIL_CONFIG = env.email_url('EMAIL_URL')
+    vars().update(EMAIL_CONFIG)
```
Comment (konstin) [inline meine_stadt_transparent/settings/__init__.py]: We can omit the default value:

```suggestion
elif "EMAIL_URL" in env:
    # If EMAIL_URL is not configured, django's SMTP defaults will be used
    EMAIL_CONFIG = env.email_url('EMAIL_URL')
    vars().update(EMAIL_CONFIG)
```

**Comment by ubergesundheit** [comment]
django-allauth also required to set `DEFAULT_FROM_EMAIL`.. When do you plan to tag a new release?

File: meine_stadt_transparent/settings/__init__.py (hunk 1 above)
Comment (ubergesundheit) [reply meine_stadt_transparent/settings/__init__.py]: Changed in https://github.com/meine-stadt-transparent/meine-stadt-transparent/pull/332/commits/958ba0581a525ff2310ec57559385a3dbbbbbc44

**Comment by konstin** [comment]
Thanks again!

> django-allauth also required to set `DEFAULT_FROM_EMAIL`.. When do you plan to tag a new release?

Yes, I'll tag this as 0.2.1.

**Pull request merged**